Makefile

```diff
@@ -1,4 +1,6 @@
 #Makefile
+JAVA11_HOME?=/Users/nenashev/Documents/tools/jdk-11/jdk-11.jdk/Contents/Home
```

Comment:
Hardcoded. Any suggestions? Or just default to the system java and use `test` target?

Reply:
On macOS at least, you can use the output of `/usr/libexec/java_home -v 11`

Reply:
or use something like `jenv` and so `$(shell jenv prefix 11)`

Reply:
what about ```readlink -e $(which java)``` or check if exists ```update-java-alternatives```, or ```update-alternatives``` or ```alternatives``` and pipe ```grep``` to the result.

Reply:
Well, non of that would work on my machine. Java 11 is not mapped to alternatives.
I tend to say that it will be just receiving JAVA_HOME and offering the rest in makefile targets

Reply:
`which java` would offer the same JVM as `$JAVA_HOME`, which is not really what we want. The JVM running the code of this project can be different from the JVM we want to test the plugin against. At least, `jenv` works fine on linux and mac.

Reply:
Would it be OK if I just rename the target to `demo-jdk11` and make the code version-agnostic for the parameter?

Reply:
should be fixed now